src/GhSync/Services/IGitHub.cs

```diff
@@ -0,0 +1,6 @@
+namespace gh_sync;
+
+public interface IGitHub
+{
+
```

Comment:
I had made this one blank, but it'd be good to fill it out some, possibly moving code from Program.cs so as to keep that file focused on being an entry point.